test/support/test_publishers.ex

```diff
@@ -0,0 +1,43 @@
+defmodule TestPublisher do
+  defmodule Default do
+    @moduledoc false
+    @behaviour GenRMQ.Publisher
+
+    def init() do
+      [
+        exchange: "gen_rmq_out_exchange",
+        uri: "amqp://guest:guest@localhost:5672",
+        app_id: :my_app_id
+      ]
+    end
+  end
+
+  defmodule WithConfirmations do
+    @moduledoc false
+    @behaviour GenRMQ.Publisher
+
+    def init() do
+      [
+        exchange: "gen_rmq_out_exchange",
+        uri: "amqp://guest:guest@localhost:5672",
+        app_id: :my_app_id,
+        activate_confirmations: true
```

Comment:
Should be `enable_confirmations: true`

Reply:
Good catch! It makes me to realize that it is hard to test confirmations :/ Any ideas?

Reply:
Hmm yeah. Only thing I can think of is to mock out AMQP.Confirm.wait_for_confirms somehow. :\